aviary/subsystems/mass/flops_based/landing_gear.py

```diff
@@ -70,21 +71,23 @@ def compute(self, inputs, outputs):
         nose_gear_scaler = inputs[Aircraft.LandingGear.NOSE_GEAR_MASS_SCALER]
         landing_weight = inputs[Aircraft.Design.TOUCHDOWN_MASS] * GRAV_ENGLISH_LBM
 
-        outputs[Aircraft.LandingGear.MAIN_GEAR_MASS] = (
+        main_gear_mass = (
             0.0117
             * landing_weight**0.95
             * main_gear_length**0.43
             * main_gear_scaler
             / GRAV_ENGLISH_LBM
         )
-        outputs[Aircraft.LandingGear.NOSE_GEAR_MASS] = (
+        nose_gear_mass = (
             0.048
             * landing_weight**0.67
             * nose_gear_length**0.43
             * nose_gear_scaler
             / GRAV_ENGLISH_LBM
         )
-
+        outputs[Aircraft.LandingGear.MAIN_GEAR_MASS] = main_gear_mass
+        outputs[Aircraft.LandingGear.NOSE_GEAR_MASS] = nose_gear_mass
+        outputs[Aircraft.LandingGear.TOTAL_MASS] = main_gear_mass + nose_gear_mass
```

Comment:
There will be problems with this if the user overrides either main or nose gear mass in the csv file, but lets aviary calculate the other one, because the replacement will happen after this calculation outside of the component. The way to resolve this is to create a new component that takes nose and main gear mass as inputs and outputs the total landinggear mass.